akka-http-core/src/main/resources/reference.conf

```diff
@@ -18,6 +18,15 @@ akka.http {
     # the request, no `Server` header will be rendered at all.
     server-header = akka-http/${akka.http.version}
 
+    # EXPERIMENTAL FEATURE: Bind using HTTP/2
```

Comment:
thought we got rid of the term experimental? This is experimental in the sense that we are uncertain of its production readiness? Could we find another word for that so that we don't make things even more confusing.